installer/scripts/omsadmin.sh

```diff
@@ -493,8 +493,12 @@ onboard_lad()
 onboard()
 {
     if [ $VERBOSE -eq 1 ]; then
+        # Mask the shared key
+        local shared_key_trunc="`echo "$SHARED_KEY" | cut -c 1-4 2> /dev/null`"
+        local shared_key_covered="`echo "$SHARED_KEY" | tr "$SHARED_KEY" "*" | cut -c 5- 2> /dev/null`"
```

Comment:
The double double quotes is a little confusing, and can encourage others to add problems.  I tend to try to use apostrophes where things are a literal, and to omit any quotation that is not necessary.  The following should work:
local shared_key_covered=\`echo -n "$SHARED_KEY" | tr "$SHARED_KEY" '*' | cut -c 5- 2>/dev/null\`

Reply:
Will fix.

Reply:
Fixed.